worlds/faxanadu/Regions.py

```diff
@@ -0,0 +1,67 @@
+from BaseClasses import Entrance, Region
+from typing import TYPE_CHECKING
+from worlds.generic.Rules import set_rule
```

Comment:
```suggestion
from BaseClasses import Region
from typing import TYPE_CHECKING
```
Remove unused imports